### DISCUSSION TITLE
GraphQL reponse for Page.single.toc: String cannot represent value: "..."

### DISCUSSION THREAD

**Discussion by rtc11** [requarks/wiki#3671] [Error / Bug Report]
**Describe the bug**
When fetching a single Page, toc returns an error and value `null` in the data response. 
My Table of Contents looks likt this:
- Method
- Preparation
- Instructions

**To Reproduce**
1. Create a new Page with at least one Heading 1 element.
2. Figure out the correct ID of this page
3. query:
```graphql
{
  pages {
    single(id: 7) {
      toc
    }
  }
}
```

**Expected behavior**
Content in field `toc` in the response other than `null`.

**Host Info (please complete the following information):**
 - OS: Docker Container (Linux)
 - Wiki.js version: 2.3.77
 - Database engine: PostgreSQL:11.7

**Comment by did** [comment]
I'm facing the same error. In my case, the page was written in Markdown in case it helps.

**Comment by SPACEX-2022** [comment]
same here

**Comment by ShynRou** [comment]
Any updates updates on this?
The error I get is: `"String cannot represent value: [{ title: \"...\", anchor: \"#...\", children: [] },...]`
Looks like the GraphQL Schema is simply wrong here.

**Comment by vuducmanh11** [comment]
Any updates :cry: 
I still get this error when query TOC of page.
